unordered-containers.cabal

```
@@ -42,6 +42,14 @@ flag debug
   description:  Enable debug support
   default:      False
 
+flag noSse42
+  description: Instruct GHC to *not* use -msse4.2. Typical use case is to allow
+               older architectures to use unordered-containers. Only relevant
+               for x86; expect performance regressions on newer architectures
+               when enabled.
+  default:     False
+
```

```suggestion
  manual:      True
```
The flag should be manual, otherwise cabal tries to set it automatically.